### ISSUE TITLE
Triplet Loss with Hard Mining

### ISSUE THREAD

**Issue by ahmedbesbes** [UKPLab/sentence-transformers#995]
Hello, 

I was wondering if it was possible to use a hard or semi hard triplet loss when you only have triplets (anchor, positive, negative) and no information about the classes each sample belongs to.

Thanks!

**Comment by nreimers** [comment]
No, that is not possible. You can use TripletLoss or MultipleNegativesRankingLoss.
